packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -420,81 +445,75 @@ class _DecoyChild extends StatefulWidget {
-    // Change the color of the child during the initial part of the decoy bounce
-    // animation. The interval was eyeballed from a physical iOS 13.1.2 device.
-    _mask = _OnOffAnimation<Color>(
-      controller: widget.controller,
-      onValue: _lightModeMaskColor,
-      offValue: _masklessColor,
-      intervalOn: 0.0,
-      intervalOff: 0.5,
-    );
-
-    final Rect midRect =  widget.beginRect!.deflate(
-      widget.beginRect!.width * (_kOpenScale - 1.0) / 2,
-    );
+    // The timming on the animation was eyeballed from the XCode iOS simulater
+    // running iOS 16.0
     _rect = TweenSequence<Rect?>(<TweenSequenceItem<Rect?>>[
       TweenSequenceItem<Rect?>(
         tween: RectTween(
           begin: widget.beginRect,
-          end: midRect,
-        ).chain(CurveTween(curve: Curves.easeInOutCubic)),
-        weight: 1.0,
+          end: widget.beginRect,
+        ).chain(CurveTween(curve: Curves.linear)),
+        weight: 3,
       ),
       TweenSequenceItem<Rect?>(
         tween: RectTween(
-          begin: midRect,
+          begin: widget.beginRect,
           end: widget.endRect,
-        ).chain(CurveTween(curve: Curves.easeOutCubic)),
-        weight: 1.0,
+        ).chain(CurveTween(curve: Curves.easeOutSine)),
+        weight: 5,
       ),
     ]).animate(widget.controller);
-    _rect.addListener(_rectListener);
-  }
 
-  // Listen to the _rect animation and vibrate when it reaches the halfway point
-  // and switches from animating down to up.
-  void _rectListener() {
-    if (widget.controller.value < 0.5) {
-      return;
-    }
-    HapticFeedback.selectionClick();
-    _rect.removeListener(_rectListener);
+    const List<BoxShadow> endBoxShadow = <BoxShadow>[
+      BoxShadow(
+        color: Color(0x40000000),
+        blurRadius: 10.0,
+        spreadRadius: 0.5,
+      )
+    ];
+
+    _boxDecoration = DecorationTween(
+      begin: BoxDecoration(
+          color: const Color(0xFFFFFFFF),
+          borderRadius: getBorderRadius(),
+          boxShadow: const <BoxShadow>[],
+      ),
+      end: BoxDecoration(
+        color: const Color(0xFFFFFFFF),
+        borderRadius: getBorderRadius(),
+        boxShadow: endBoxShadow,
+      ),
+    ).animate(widget.controller);
   }
 
   @override
   void dispose() {
-    _rect.removeListener(_rectListener);
     super.dispose();
   }
 
   Widget _buildAnimation(BuildContext context, Widget? child) {
-    final Color color = widget.controller.status == AnimationStatus.reverse
-      ? _masklessColor
-      : _mask.value;
     return Positioned.fromRect(
       rect: _rect.value!,
-      child: ShaderMask(
-        key: _childGlobalKey,
-        shaderCallback: (Rect bounds) {
-          return LinearGradient(
-            begin: Alignment.topLeft,
-            end: Alignment.bottomRight,
-            colors: <Color>[color, color],
-          ).createShader(bounds);
-        },
+      child: Container(
+        key: const Key('context-decoy-container'),
```

Comment:
Is the only point of this Key to be able to find it in the tests?  If so, you can probably get around it by getting fancier with Finder.  For example, you can find the private class _DecoyChild like this if it helps:

```dart
  find.descendant(
    of: find.byType(CupertinoContextMenu),
    matching: find.byWidgetPredicate((Widget w) => '${w.runtimeType}' == '_DecoyChild'),
  );
```

Reply:
Yes, I just needed to find it, and I lost the ShaderMask that was used before. That _DecoyChild Finder is way better, and I was looking for something like that. Thank you.